docs/content/docs/dev/table/sql/create.md

```diff
@@ -513,6 +513,46 @@ If you provide no like options, `INCLUDING ALL OVERWRITING OPTIONS` will be used
 
 **NOTE** The `source_table` can be a compound identifier. Thus, it can be a table from a different catalog or database: e.g. `my_catalog.my_db.MyTable` specifies table `MyTable` from catalog `MyCatalog` and database `my_db`; `my_db.MyTable` specifies table `MyTable` from current catalog and database `my_db`.
 
+### `AS`
+
+The AS clause is a variation of SQL features (Feature T172, “AS subquery clause in table definition”).
+The clause can be used to create a table based on the given query expression. It will be more user-friendly and reduce the cost for user to manually spell complex table creation statements.
```

Comment:
Reference:
1. https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-develop-ctas#create-table-as-select
2. https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-CreateTableAsSelect(CTAS)


Tables can also be created and populated by the results of a query in one create-table-as-select (CTAS) statement. CTAS is the simplest and fastest way to create and insert data into a table with a single command.

There are two parts in CTAS, the SELECT part can be any [SELECT statement](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/queries/overview/) supported by Flink SQL. The CREATE part of the CTAS takes the resulting schema from the SELECT part and creates the target table with other table properties such as the connector and URL. Similar to CREATE TABLE, CTAS requires the required options of the corresponding connector must be specified in WITH clause.